release.sh

```diff
@@ -107,7 +107,7 @@ export GITHUB_TOKEN=""
 # Convenience function to run the hub tool.
 # Parameters: $1..$n - arguments to hub.
 function hub_tool() {
-  gorun github.com/github/hub@v2.14.2 "$@"
+  go run github.com/github/hub@v2.14.2 "$@"
```

Comment:
```suggestion
  go_run github.com/github/hub@v2.14.2 "$@"
```
There is a `go_run` func, that should be used here: https://github.com/knative/hack/blob/e81d63ab688fde4f41078224a05325bf4124a6c1/library.sh#L555